packages/proton-bridge/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ProtonMail/proton-bridge
+TERMUX_PKG_DESCRIPTION="ProtonMail Bridge application"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=1.4.5
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+
```

Comment:
Feel free to add yourself as TERMUX_PKG_MAINTAINER as well!

Reply:
I pushed a new commit with this.